pacu/modules/organizations__assume_role/main.py

```diff
@@ -0,0 +1,150 @@
+
+    # A list of AWS services that the module utilizes during its execution.
+    'services': ['Organizations'],
+
+    # For prerequisite modules, try and see if any existing modules return the
+    # data that is required for your module before writing that code yourself;
+    # that way, session data can stay separated and modular.
+    'prerequisite_modules': ['organizations__enum'],
+
+    # External resources that the module depends on. Valid options are either
+    # a GitHub URL (must end in .git), or a single file URL.
+    'external_dependencies': [],
+
+    # Module arguments to autocomplete when the user hits tab.
+    'arguments_to_autocomplete': ['accounts','accounts-file','roles','roles-file'],
+}
+
+parser = argparse.ArgumentParser(add_help=False, description=module_info['description'])
+
+parser.add_argument('--accounts', required=False, nargs='+', default=None,  help='Pass in a list of accounts to try assuming, --accounts 1 2 3')
+parser.add_argument('--accounts-file', required=False, nargs=1, default=None, help="Pass in a filename containing a list of accounts to try assuming")
+parser.add_argument('--roles', required = False, nargs = '+', default=None,  help='Pass in a list of roles to try assuming. If non are specified uses default OrganizationAccountAccessRole, --roles a b c')
+parser.add_argument('--roles-file', required=False, nargs=1, default=None, help='Pass in a filename containing a list of roles to try assuming')
+#parser.add_argument('--create-sessions', required=False, action="store_true", help='Will automatically create a session for every successful AssumeRole operation. Can check all of them after with swap_keys command.')
+
+roles_list = []
+
+def assume_role(args, session, role_arn):
+    global roles_list
+    sts = boto3.client('sts')
+    try:
+        resp = sts.assume_role(
+            RoleArn=role_arn,
+            RoleSessionName=''.join(random.choice(string.ascii_letters) for _ in range(20)),
+        )
+        roles_list.append(role_arn)
+        print("SUCCESS: " + role_arn)
+        
+        # See if I can create a key set automatically in background
+        #if args.create_sessions:
+
+        #    print("CREATING KEYS FOR: " + role_arn)
+        #    new_key_name = f"{resp['AssumedRoleUser']['Arn']}"
+        #    aws.create_session
+        #    session.set_keys(
+        #        key_alias=new_key_name,
+        #        access_key_id=resp['Credentials']['AccessKeyId'],
+        #        secret_access_key=resp['Credentials']['SecretAccessKey'],
+        #        session_token=resp['Credentials']['SessionToken'],
+        #    )
+        #else:
+        #    pass
+
+    except ClientError as error:
+        code = error.response['Error']['Code']
+        if code == 'AccessDenied':
+            print("FAILURE: "+role_arn)
+        else:
+            print('(DEFAULT FAILURE: {}'.format(code))
```

Comment:
There is an extra parenthesis in the print statement: '(DEFAULT ...'

Reply:
Fixed